lib/shortcodes/property-attribute.php

```diff
@@ -70,7 +70,13 @@ public function __construct() {
                 'name' => __( 'Strip Tags', ud_get_wp_property()->domain ),
                 'description' => __( 'Strip tags', ud_get_wp_property()->domain ),
                 'default' => ''
-              )
+              ),
+              'show_taxonomy_label'=> array(
+                'name' => __( 'Show Taxanomy Label', ud_get_wp_property()->domain ),
+                'description' => __( 'True/False : Show/Hide taxanomy label', ud_get_wp_property()->domain ),
+                'type' => 'text',
+                'default' => true
```

Comment:
@raj-uimatic I don't think settings should be on by default as a general rule. Reason being is we have to consider thousands of people who already have a site setup, perhaps using this shortcode. They may have styled their themes to look in a particular way. All of a sudden after they upgrade some new element will start showing up. (At least that's my understanding based on code).

So, when it comes to rolling out upgrades to lots of production sites, it's better to play it safe and expect people to explicitly enable new functionality.